examples/stable_diffusion_v2/ldm/models/diffusion/dpm_solver/sampler.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, model, algorithm_type="dpmsolver", **kwargs):
         self.algorithm_type = algorithm_type
         self.register_buffer("alphas_cumprod", model.alphas_cumprod)
         self.noise_schedule = NoiseScheduleVP("discrete", alphas_cumprod=self.alphas_cumprod)
+        self.model_type = kwargs.get("model_type", "noise")
```

Comment:
Suggest follow the naming in diffuser. `model_type` -> `prediction_type`
ref: https://huggingface.co/docs/diffusers/api/schedulers/ddpm#diffusers.DDPMScheduler.prediction_type